scripts/report_scripts/index.Rmd

```diff
@@ -882,12 +882,12 @@ cat(
 )
 ```
 
-```{r table_ample_quality_score}
+```{r table_ample_quality_score, warning=TRUE}
 if (!(length(coverages.df) == 0)) {
```

Comment:
This is not your change, but it jumps out at me, so I must mention it.  I hold deep contempt for this idiom, because it is so very crude.  There are two things wrong with it:

- it's inverted; the condition should be inverted and the branches of the `if` swapped
- it's crude because it doesn't express clearly what we're testing.  We want some higher level method of checking if the coverages.df is empty; we don't *actually* care about it's length, nor do we care about comparing it with zero.  We just want to know if we have values here.

It's also crude because coverages.df is not an explicit argument to any of the code in the first branch, so it's not clear why this test even matters.

(Sorry, this is peak nit-picking.  It's my only talent.)

Reply:
Well good news, that will be changed in commit f5f01e0 on #133 ^^